File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/icourtesy-batch_item.ts




export interface ICourtesyBatch_Item {
  
	CourtesyBatchID: number;
	VettingBatchID: number;
	VettingTypeID: number;
	VettingType: string;
	VettingBatchNotes: string;
	AssignedToAppUserID?: number;
	AssignedToAppUserName: string;
	ResultsSubmittedDate?: Date;
	ResultsSubmittedByAppUserID?: number;
	ResultsSubmittedByAppUserName: string;

}

File: GTTS/INL.VettingService.Models/ICourtesyBatch_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public interface ICourtesyBatch_Item
    {
        long CourtesyBatchID { get; set; }
        long VettingBatchID { get; set; }
        int VettingTypeID { get; set; }
        string VettingType { get; set; }
        string VettingBatchNotes { get; set; }
        int? AssignedToAppUserID { get; set; }
        string AssignedToAppUserName { get; set; }
        DateTime? ResultsSubmittedDate { get; set; }
        int? ResultsSubmittedByAppUserID { get; set; }
        string ResultsSubmittedByAppUserName { get; set; }
    }
}
